CRITICAL - 2020-08-31 00:17:33 --> Class Home does not exist
#0 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\Routes.php(107): ReflectionMethod->__construct('Home', '_remap')
#1 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(307): CodeIgniter\Debug\Toolbar\Collectors\Routes->display()
#2 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\Debug\Toolbar.php(128): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#3 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\Debug\Toolbar.php(353): CodeIgniter\Debug\Toolbar->run(1598851053.4301, 0.2066, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#4 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(78): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#5 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\Filters\Filters.php(199): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#6 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\CodeIgniter.php(422): CodeIgniter\Filters\Filters->run('', 'after')
#7 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ApotekAPP\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-08-31 00:58:23 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ApotekAPP\app\Views\pegawai\formtambahbanyak.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 17, Array)
#1 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('pegawai/formtam...', Array, true)
#3 C:\xampp\htdocs\ApotekAPP\app\Controllers\Pegawai.php(166): view('pegawai/formtam...')
#4 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Pegawai->formtambahbanyak()
#5 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#6 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ApotekAPP\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-08-31 01:00:49 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ApotekAPP\app\Views\pegawai\formtambahbanyak.php(16): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 16, Array)
#1 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('pegawai/formtam...', Array, true)
#3 C:\xampp\htdocs\ApotekAPP\app\Controllers\Pegawai.php(166): view('pegawai/formtam...')
#4 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Pegawai->formtambahbanyak()
#5 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pegawai))
#6 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ApotekAPP\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-08-31 04:20:15 --> Invalid file: layout/under.php
#0 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('layout/under.ph...')
#1 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('layout/under', Array, true)
#2 C:\xampp\htdocs\ApotekAPP\app\Controllers\Pembelian.php(9): view('layout/under')
#3 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Pembelian->index()
#4 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pembelian))
#5 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ApotekAPP\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2020-08-31 05:58:57 --> Unable to connect to the database.
#0 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\ApotekAPP\app\Controllers\Login.php(92): CodeIgniter\Database\BaseConnection->query('SELECT users.`i...')
#2 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Login->cekuser()
#3 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#4 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\ApotekAPP\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2020-08-31 05:59:01 --> Unable to connect to the database.
#0 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\ApotekAPP\app\Controllers\Login.php(92): CodeIgniter\Database\BaseConnection->query('SELECT users.`i...')
#2 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Login->cekuser()
#3 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#4 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\ApotekAPP\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2020-08-31 05:59:47 --> Invalid file: layout/under.php
#0 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('layout/under.ph...')
#1 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('layout/under', Array, true)
#2 C:\xampp\htdocs\ApotekAPP\app\Controllers\Pembelian.php(9): view('layout/under')
#3 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Pembelian->index()
#4 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pembelian))
#5 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ApotekAPP\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2020-08-31 05:59:50 --> Invalid file: layout/under.php
#0 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('layout/under.ph...')
#1 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('layout/under', Array, true)
#2 C:\xampp\htdocs\ApotekAPP\app\Controllers\Pembelian.php(9): view('layout/under')
#3 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Pembelian->index()
#4 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pembelian))
#5 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ApotekAPP\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\ApotekAPP\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#8 {main}
